api/src/providers/apiNodeProvider.ts

```diff
@@ -339,7 +339,9 @@ export async function getDeployment(owner: string, dseq: string) {
     include: [{ model: ProviderAttribute }]
   });
 
-  const aktPrice = getAktMarketData()?.price;
+  const marketData = await getMarketData();
```

Comment:
Isn't this doing a query to coingecko every time?

Reply:
Yes indeed I'll add caching